index.html

```diff
@@ -154,30 +167,33 @@ <h5>LET'S CONNECT</h5>
       </div>
       <button type="button" id="aboutMyselfBotton">Get My Resume</button>
   </div>
+  <div class="desktopVersion">
   <div class="languageAndArrowContainer">
-    <span>Languages</span>
+    <span class="span-language lang">Languages</span>
     <img src="assets/downSideArrow.svg" alt="">
   </div>
   <div class="languagesContainer">
-    <ul>
-      <li class="languages"><img src="assets/javascriptAsset.svg" alt="jsicon"> <span id="js">JavaScript</span></li>
-      <li class="languages"><img src="assets/htmlAsset.svg" alt="htmlicon"><span id="html">HTML</span></li>
-      <li class="languages"><img src="assets/cssAsset.svg" alt="cssicon"><span id="css">CSS</span></li>
+    <ul class="container-desktop">
+      <li class="languages"><img class="languages-img" src="assets/javascriptAsset.svg" alt="jsicon"> <span id="js">JavaScript</span></li>
+      <li class="languages"><img class="languages-img" src="assets/htmlAsset.svg" alt="htmlicon"><span id="html">HTML</span></li>
+      <li class="languages"><img class="languages-img" src="assets/cssAsset.svg" alt="cssicon"><span id="css">CSS</span></li>
```

Comment:
- [ ] Kindly as requested by the previous reviewer consider aligning the technologies names  listed in the About me  section to the left as shown in the Figma design given to make your design an exact duplicate of the given one 
- your design 👇 
<img width="673" alt="Screen Shot 2023-06-17 at 20 59 05" src="https://github.com/fmalgarini/My-portfolio-Project/assets/60146030/262cf777-c99d-458c-a990-8fb5d9109726">

- Figma design 👇 
<img width="536" alt="Screen Shot 2023-06-17 at 21 00 30" src="https://github.com/fmalgarini/My-portfolio-Project/assets/60146030/2bf36c00-62e6-4c3d-a87e-3ed1f4f1aac8">